src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,559 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Preload video and audio for faster playback.
+
+{# wf_published_on: 2017-07-05 #}
+{# wf_updated_on: 2017-07-05 #}
+
+# Fast Playback with Video Preload {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+<video controls controlsList="nodownload" muted playsinline style="width: 100%">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.webm#t=0.7"
+          type="video/webm">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/videos/video-preload-hero.mp4#t=1"
+          type="video/mp4">
+</video>
+
+Faster playback means more people watching your video. That's a known fact.
+I'll explore in this article different techniques you can use to accelerate
+your media playback by actively preloading resources depending on your use
+case.
+
+Note: Unless specified otherwise, this article also applies to the audio element.
+
+### TL;DR {: .hide-from-toc }
+
+<table>
+  <tbody>
+    <tr>
+      <th></th>
+      <th>
+Pros
+      </th>
+      <th>
+Cons
+      </th>
+    </tr>
+    <tr>
+      <td rowspan=3 style="white-space: nowrap">
+<a href="#video_preload_attribute">Video preload attribute</a>
+      </td>
+      <td rowspan=3>
+Simple to use for an unique file hosted on a web server
+      </td>
+      <td>
+Browser may completely ignore the attribute
+      </td>
+    <tr>
+      <td>
+Resource fetching starts when HTML document has been completely loaded and
+parsed
+      </td>
+    </tr>
+    <tr>
+      <td>
+Doesn't work with MSE
```

Comment:
I've tried something else... what do you think of https://github.com/google/WebFundamentals/pull/4744/commits/02439c38d3d4115bedab69952316fe7935bfe000